Hyper/GameEntities/AstronautBot.cs

```diff
@@ -146,6 +148,48 @@ private void UpdateBotGoalsWhenHostile(Simulation simulation, float time, Scene
             movementDirection);
     }
 
+    public override void ContactCallback(ContactInfo collisionInfo, SimulationMembers simulationMembers)
```

Comment:
why is this overridden with the same exact thing?

Reply:
There is 1 extra line that sets the bot to hostile when hit with a projectile

Reply:
It's line 173

Reply:
Ok